src/views/ChuckView.vue

```diff
@@ -0,0 +1,208 @@
+<template>
+  <div class="user">
+    <h1>Chuck facts</h1>
+    <p>In archive {{archive.length}}</p>
+    <p v-show="avVar">Average rating: {{ avVar }}</p>
+    <div v-for="fact in factsArray" class="fact">
+      <div v-if="doesRatingExist(fact.id)">
+        <div class="rating">{{ ratings[fact.id] }}</div>
+        <button class="remove-rating" @click="removeRating(fact.id)">remove rting</button>
```

Comment:
Please correct the typo and capitalize the user action caption:
```suggestion
        <button class="remove-rating" @click="removeRating(fact.id)">Remove rating</button>
```
If the app is going to be internationalized, consider using gettext for providing translation interpolation.